Jasmine/Common/Secrets.swift

```diff
@@ -0,0 +1,9 @@
+/// Contains api keys and other files that should not be checked into repo
+enum Secrets {
```

Comment:
It is checked though? :o

Maybe use `Secrets.swift.default` instead as a skeleton?